### DISCUSSION TITLE
Add more design languages as option in settings - Material Design

### DISCUSSION THREAD

**Discussion by inson1** [clementwzk/OpenCalc#391] [Ideas]
"Why Calculator++? Why not OpenCalc?"

"Because I hate Material You."

"What design language Calculator++ uses?"

"Classic Material Design, with no space between buttons."

from reddit

**Comment by inson1** [comment]
how big are all the icons? I hope it isnt some big % but I think they are :/

**Reply by clementwzk** [reply]
they are vectors

**Reply by inson1** [reply]
@Darkempire78 I meant size :) its 2 MB APK

**Reply by clementwzk** [reply]
no idea x)

**Comment by clementwzk** [comment]
Most of users use Material design You or will use it in the future. I will not make an other design because I do not want to maintain it especially if it is not really used. We can make a poll to confirm that and share it on open source reddits to get answers
